PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -0,0 +1,10 @@
+## Version 20.1.0
+_Date TBD_
+
+### New Features
+* Update invalid input fields
```

Comment:
This is changing some errors to warnings, or warnings to errors, and maybe fixing some typos, right? RELEASE NOTES are meant to be read by the public so should have a description of the changes that a layperson would understand. Also shouldn't have an empty Bugs Fixed section